bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/thing/sink.xml

```diff
@@ -21,6 +21,18 @@
 				<label>Name</label>
 				<description>The name of one specific device.</description>
 			</parameter>
+			<parameter name="activateSimpleProtocolSink" type="boolean">
+				<label>Sink by simple-protocol-tcp</label>
+				<description>Activation of a corresponding sink in openHAB (needs module-simple-protocol-tcp loaded on the server)</description>
+				<default>false</default>
+				<required>false</required>
+			</parameter>
+			<parameter name="simpleProtocolSinkPort" type="integer">
+				<label>Simple protocol port</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Thank you for you review. I will reread the coding guidelines for the next time.
I made all the aforementioned adjustement.
(I'm not used to this review process, don't hesitate to tell me if I didn't do something correctly/politly)

Reply:
You've done a good job so far :)